src/pybind/mgr/cephadm/services/cephadmservice.py

```diff
@@ -685,6 +685,7 @@ def prepare_create(self, daemon_spec: CephadmDaemonDeploySpec) -> CephadmDaemonD
         if ports:
             daemon_spec.ports = ports
 
+        daemon_spec.ports.append(self.mgr.service_discovery_port)
```

Comment:
this looks like it's universally adding this port for every daemon we deploy? I would think we only want to do this for the mgr daemon.

Reply:
> this looks like it's universally adding this port for every daemon we deploy? I would think we only want to do this for the mgr daemon.

This is done as part of the `class MgrService(CephService)`, would it affect to other daemons other than mgr?

Reply:
no, I think it's fine. Not sure what I was thinking when I wrote the first comment.